lib/ch/row_binary.ex

```diff
@@ -504,6 +504,12 @@ defmodule Ch.RowBinary do
         <<d::32-little-signed, bin::bytes>> = bin
         decode_rows(types_rest, bin, [Date.add(@epoch_date, d) | row], rows, types)
 
+      :datetime -> 
+      <<s::32-little, bin::bytes>> = bin
+      dt =NaiveDateTime.add(@epoch_naive_datetime, s)
+      decode_rows(types_rest, bin, [dt | row], rows, types)
```

Comment:
Sorry for making things complicated in the codebase, but I think `:datetime` should never match. When we receive RowBinaryWithNamesAndTypes, DateTime would be decoded to `{:datetime, _tz = nil}` and when we pass the types explicitely to decode RowBinary, `:datetime` would be turned into `{:datetime, _tz = nil}`. Or at least it should :)

Reply:
Thx. I haven't realised I was using an out of date remote. The decoding works it seems.